lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+from flatten_json import flatten
+
+
+def lusid_respose_to_pd(lusid_response):
```

Comment:
should this be `lusid_respose_to_data_frame` rather than `pd`? (adopting whatever convention we've already used for DataFrame in method names)

Reply:
Agreed. Based on comments above, I wont return a pd.Series. Therefore **lusid_response_to_data_frame** is better name.